app-modules/core/database/seeders/RoleSeeder.php

```
@@ -0,0 +1,20 @@
+use Spatie\Permission\Models\Role;
+
+class RoleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        Role::firstOrCreate([
+            'name' => RoleEnum::Admin,
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Spatie\Permission\Models\Role' in method 'run'. (undefined)

(StaticAccess)